src/meta/Dimensions.ts

```diff
@@ -0,0 +1,63 @@
+import { Base } from './Base';
+
+export interface DimensionResults {
```

Comment:
I could see it either way. on the one hand we've already made it nicer because you don't have to remember where the properties are actually coming from, you just have a big property bag. On the other hand, it might be nice to separate them out into groups like that.